letsencrypt/cli.py

```diff
@@ -351,7 +351,8 @@ def diagnose_configurator_problem(cfg_type, requested, plugins):
             msg = "The requested {0} plugin does not appear to be installed".format(requested)
         else:
             msg = ("The {0} plugin is not working; there may be problems with "
-                   "your existing configuration").format(requested)
+                   "your existing configuration.\nThe error was: {1}"
+                   .format(requested, plugins[requested].problem))
```

Comment:
I thought about displaying the exception type, but when I tried it out it produced a somewhat confusing result.  For instance with jsha's example of the problem from #971, we get:

```
pde@damoclid:~/letsencrypt/letsencrypt$ sudo venv/bin/letsencrypt  --email foo@example.com --server http://localhost:4000/directory  --text --agree-dev-preview --agree-tos -d foo.com --config-dir $TMP --work-dir $TMP --logs-dir  $TMP --dvsni-port 5001 --nginx-server-root $TMP -a nginx  auth 
The nginx plugin is not working; there may be problems with your existing configuration.
The error was: NoInstallationError('Could not find configuration root',)
```
Arguably that's not really a "NoInstallationError".  But your example suggests that it might be worth the price, so I'll make the change.